MaxText/configs/base.yml

```diff
@@ -569,8 +569,8 @@ eval_steps: -1  # run this number of steps for eval, recommend setting this to p
 target_eval_loss: 0.  # early stop once reaching target eval_loss
 
 # Goodput parameters
-enable_goodput_recording: True
-monitor_goodput: True
+enable_goodput_recording: False
+monitor_goodput: False
```

Comment:
I'm not yet fully sure if we should disable Goodput by default, especially without first evaluating alternatives based on the symptoms we are seeing.

If we do decide to disable Goodput by default, the documentation needs to be updated as well.

Reply:
Specifically [this](https://github.com/AI-Hypercomputer/maxtext/blob/main/getting_started/Monitor_Goodput.md#how-to-monitor-goodput-and-badput) section that calls out it is enabled by default.